bsdlib/doc/security_tags.rst

```diff
@@ -0,0 +1,51 @@
+.. _security_tags:
+
+Security tags
+#############
+
+The modem on an nRF9160 device is equipped with a full IPv4/IPv6 stack and a DTLS/TLS engine.
```

Comment:
DTLS / TLS are not engines, they are protocols.

Reply:
"... with a full IPv4/IPv6 stack that supports DTLS/TLS protocols"?

Reply:
Yes, could omit `protocols` altogether e.g. `[...] with TLS/DTLS support`.